internal/pkg/install/install.go

```diff
@@ -83,7 +91,7 @@ func Install(args string, data *userdata.UserData) (err error) {
 
 			switch {
 			case strings.HasSuffix(sourceFile.Name(), ".tar") || strings.HasSuffix(sourceFile.Name(), ".tar.gz"):
-				log.Printf("Extracting %s to %s\n", sourceFile.Name(), dest)
+				log.Printf("extracting %s to %s\n", sourceFile.Name(), dest)
```

Comment:
Most of the rest of the codebase uses lowercase. Figured I'd align this with that convention and we can discuss a style we agree on and refactor later.